pipelines/LSSTCam/quickLook.yaml

```diff
@@ -22,47 +22,14 @@ tasks:
       doDefect: false
       doDark: false
       doFlat: false
-
-  # TO DO: Replace with calibrateImage in a way that we can use the
-  # preSources downstream.
-  characterizeImage:
-    class: lsst.pipe.tasks.characterizeImage.CharacterizeImageTask
-    config:
-      doApCorr: false
-      doDeblend: false
-  calibrate:
-    class: lsst.pipe.tasks.calibrate.CalibrateTask
-    config:
-      python: |
-          config.measurement.plugins.names = ['base_CircularApertureFlux',
-                                              'base_PsfFlux',
-                                              'base_LocalBackground',
-                                              'base_SdssCentroid',
-                                              'base_SdssShape',
-                                              'base_Variance',
-                                              'base_Jacobian',
-                                              'base_PixelFlags',
-                                              'base_GaussianFlux',
-                                              'base_SkyCoord',
-                                              'base_FPPosition',
-                                              'base_ClassificationSizeExtendedness',
-                                              'base_CompensatedTophatFlux',
-                                              ]
-          config.measurement.slots.shape = "base_SdssShape"
-          config.measurement.slots.psfShape = "base_SdssShape_psf"
-          config.measurement.plugins['base_PixelFlags'].masksFpAnywhere.remove('STREAK')
-          config.measurement.plugins['base_PixelFlags'].masksFpCenter.remove('STREAK')
-          config.doApCorr = False
-          config.doDeblend = False
-          config.astrometry.sourceSelector["science"].doRequirePrimary = False
-          config.astrometry.sourceSelector["science"].doIsolated = False
+  calibrateImage:
+    class: lsst.pipe.tasks.calibrateImage.CalibrateImageTask
```

Comment:
I'll have to look into this into more detail on [DM-47632](https://rubinobs.atlassian.net/browse/DM-47632), where I'll switch ComCam `quickLook` (the only really important variant of `quickLook`) over to `calibrateImage`.  For now:
 - I'm not sure we should even have a version of LSSTCam `quickLook` that tries to do any on-sky processing, because it'll surely get out of date before it becomes useful, but I assume Merlin added it because it was useful for something.  I hope the [DM-47320](https://rubinobs.atlassian.net/browse/DM-47320) refactor will land first and basically reset it to a reference to a base pipeline shared with the ComCam one.
 - `calibateImage` and the `quickLook` version of `characterizeImage` and `calibrate` _have_ been optimized (independently) for very similar goals - fast, no frills processing, in Prompt Processing and Rapid Analysis, respectively.  So it's at least plausible there will not be an need to change the default configurations.